consensus/hotstuff/voteaggregator/interfaces.go

```diff
@@ -0,0 +1,31 @@
+package voteaggregator
+
+import (
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// VoteCollectors is an interface which allows VoteAggregator to interact with collectors structured by
+// view and blockID.
+// Implementations of this interface are responsible for state transitions of `VoteCollector`s and pruning of
+// stale and outdated collectors by view.
+type VoteCollectors interface {
+	// GetOrCreateCollector is used for getting hotstuff.VoteCollector, calling this function for first time
+	// will create a new collector.
+	// collector is indexed by blockID and view
+	// It returns the vote collector state machine, true and nil if found,
+	// It returns (nil, false, nil) if not found
```

Comment:
There is a typo here, it should return `(collector, true, nil)` or `(collector, false, nil)` in case of success. Nil collector can be only in case of error